imports/ui/explorer-components/sort-items.js

```diff
@@ -0,0 +1,52 @@
+export const SORT_BY = {
+  DATE_DESCENDING: 0,
+  DATE_ASCENDING: 1,
+  NAME_ASCENDING: 2,
+  NAME_DESCENDING: 3,
+  LATEST_UPDATE: 4,
+  OLDEST_UPDATE: 5
+}
+
+export const labels = [
+  [SORT_BY.DATE_DESCENDING, {category: 'Created - Latest', selected: 'Created ↓'}],
+  [SORT_BY.DATE_ASCENDING, {category: 'Created - Oldest', selected: ' Created ↑'}],
+  [SORT_BY.NAME_ASCENDING, {category: 'Name (A to Z)', selected: 'Name ↑'}],
+  [SORT_BY.NAME_DESCENDING, {category: 'Name (Z to A)', selected: 'Name ↓'}]
+]
+
+export const sorters = {
+  [SORT_BY.DATE_DESCENDING]: (a, b) => {
+    const latestCaseAunit = a.items && a.items[0].creation_time
+    const latestCaseBunit = b.items && b.items[0].creation_time
+    const dateA = Date.parse(latestCaseAunit && latestCaseAunit) || Date.parse(a.creation_time)
+    const dateB = Date.parse(latestCaseBunit && latestCaseBunit) || Date.parse(b.creation_time)
+    return dateB - dateA
+  },
+  [SORT_BY.DATE_ASCENDING]: (a, b) => {
+    const oldestCaseAunit = a.items && a.items[0].creation_time
+    const oldestCaseBunit = b.items && b.items[0].creation_time
+    const dateA = Date.parse(oldestCaseAunit && oldestCaseAunit) || Date.parse(a.creation_time)
+    const dateB = Date.parse(oldestCaseBunit && oldestCaseBunit) || Date.parse(b.creation_time)
+    return dateA - dateB
+  },
+  [SORT_BY.NAME_ASCENDING]: (a, b) => {
+    const nameA = a.unitTitle || a.title || a.name
```

Comment:
I don't like that there are specific item attributes here. It should be more generic. Since it'll be slightly harder to refactor quickly. Let's just leave it as is for now

Reply:
👍👍